src/lib/constants.ts

```diff
@@ -0,0 +1,10 @@
+export const ERROR_MESSAGE = {
+    NO_VAULTS_FOUND_BY_TITLE: "No Vaults found with title",
+    MULTIPLE_VAULTS_FOUND_BY_TITLE: "Found multiple Vaults with given title. Provide a more specific Item title",
```

Comment:
Change to `Prove a more specific Vault title`

Reply:
done